cmd.go

```diff
@@ -257,7 +257,7 @@ func handleCommandError(c Command, ctx *Context, err error, f *gnuflag.FlagSet)
 	case ErrSilent:
 		return 2, true
 	default:
-		fmt.Fprintf(ctx.Stderr, "error: %v\n", err)
+		logger.Errorf("%v\n", err)
```

Comment:
I'm not 100% sure if "juju command" includes Error messages by default. I know we *don't* report content from the Log stream unless you supply something like --debug or --verbose.

I'm fine moving the content to the logger interface as long as it actually still is always reported. (having 'juju foo' fail but suppress the error message by-default would be a very bad user experience.)